src/channeldef.ts

```diff
@@ -198,6 +199,72 @@ export type TextDef<F extends Field> =
   | FieldOrDatumDefWithCondition<StringDatumDef<F>, Text>
   | ValueDefWithCondition<StringFieldDef<F>, Text>;
 
+/**
+ * Position of a label relative to its base mark's position, which includes anchor direction (anchor) and distance (offset).
+ */
+export type LabelPosition = {
+  /**
+   * distance of a label from its base mark.
```

Comment:
Distance -> D should be a capital letter.   
Please revise other docs comment so that all sentences begin with a capital letter.